editor/editor_resource_picker.h

```diff
@@ -134,6 +136,8 @@ class EditorResourcePicker : public HBoxContainer {
 	bool is_toggle_mode() const;
 	void set_toggle_pressed(bool p_pressed);
 
+	void set_owner(Object *p_object);
```

Comment:
This shadows a method in `Node`, you need to name it something else, you shouldn't use this method to achieve this